prow/pod-utils/downwardapi/jobspec_test.go

```diff
@@ -23,6 +23,13 @@ import (
 	prowapi "k8s.io/test-infra/prow/apis/prowjobs/v1"
 )
 
+func decorateWithGitEnvs(envs map[string]string) map[string]string {
```

Comment:
You already modify the `envs`. There is no need to return as well.

Reply:
If I want to decorate I have to do it this way, to have an in-place new map. In C++ it will be optimized by a compiler, I suspect golang is no different?